docs/examples/lldp.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: nmstate.io/v1
```

Comment:
we should test this example

Reply:
we can do so once we have a release with the lldp.yaml, otherwise it break upgrade

Reply:
why we cannot do it on this PR ? there is  example test suite at test/e2e